deploy/k8s-osc-ccm/values.yaml

```diff
@@ -25,4 +25,11 @@ podLabels: {}
 # -- Assign Pod to Nodes (see [kubernetes doc](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/))
 nodeSelector: {}
 # -- Pod tolerations (see [kubernetes doc](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/))
-tolerations: []
\ No newline at end of file
+tolerations: []
+
+# -- HTTP proxy configuration example 
```

Comment:
We use helm docs.
In order to have a clear and well written docs, Can you set please values and comments in this form:
```
# -- HTTP proxy configuration example 
caBundle:
# -- Explain caBundle Name value
  name: ''
# -- Explain caBundle Key value
  key: ''
# -- Explain httpsProxy value
httpsProxy: ''
# -- Explain noProxy value
noProxy: ''
```